critiquebrainz/ws/review/views.py

```diff
@@ -387,6 +389,12 @@ def review_list_handler():
             'reviews': reviews,
             'count': count,
         }, namespace=REVIEW_CACHE_NAMESPACE)
+    if review_type:
```

Comment:
I think that we should pass this parameter into `db_review.list_reviews`, instead of post-filtering it in python.

Keep in mind that reviews can contain either a text review, or a numerical rating, or both. This means that if you want to filter ratings, we should explicitly check `where rating is not null`. Note that in your check here, you would remove all items that have a rating of 0!